index.ts

```diff
@@ -45,14 +47,6 @@ abstract class NgComponent<Props, State> {
     }
   }
 
-  $onInit() {
-    this.componentWillMount()
-  }
-
-  $postLink() {
-    this.componentDidMount()
```

Comment:
It seems like this should stay in $postLink, no? See https://github.com/angular/angular.js/blob/38f8c97af74649ce224b6dd45f433cc665acfbfb/src/ng/compile.js#L2719